webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala

```diff
@@ -1382,6 +1382,17 @@ object ValueContentV2 extends ValueContentReaderV2[ValueContentV2] {
             log = log
           )
 
+        case OntologyConstants.KnoraApiV2Complex.BundleFileValue =>
+          BundleFileValueContentV2.fromJsonLDObject(
+            jsonLDObject = jsonLDObject,
+            requestingUser = requestingUser,
+            responderManager = responderManager,
+            storeManager = storeManager,
+            featureFactoryConfig = featureFactoryConfig,
+            settings = settings,
+            log = log
```

Comment:
If `name = name` you can probably replace it just with `name`.

Reply:
in principal, yes. but it's the identical in a number of calls right above that one, so I'd have to refactor all of those too for consistency. And that would unnecessarily bloat this PR